src/Compiler/Microsoft.CodeAnalysis.Razor/test/TagHelperDescriptorFactoryTagHelpers.cs

```diff
@@ -3,462 +3,471 @@
-}
+        [HtmlAttributeName("invalid-name7")]
+        public IDictionary<string, object> GetOnlyDictionaryPropertyWithAttributeName { get; }
+    }
 
-public class DictionarySubclass : Dictionary<string, string>
-{
-}
+    public class DictionarySubclass : Dictionary<string, string>
+    {
+    }
 
-public class DictionaryWithoutParameterlessConstructor : Dictionary<string, string>
-{
-    public DictionaryWithoutParameterlessConstructor(int count)
-        : base()
+    public class DictionaryWithoutParameterlessConstructor : Dictionary<string, string>
     {
+        public DictionaryWithoutParameterlessConstructor(int count)
+            : base()
+        {
+        }
     }
-}
 
-public class DictionaryOfIntSubclass : Dictionary<int, string>
-{
-}
+    public class DictionaryOfIntSubclass : Dictionary<int, string>
+    {
+    }
 
-public class GenericDictionarySubclass<TValue> : Dictionary<string, TValue>
-{
-}
+    public class GenericDictionarySubclass<TValue> : Dictionary<string, TValue>
+    {
+    }
 
-[OutputElementHint("strong")]
-public class OutputElementHintTagHelper : TagHelper
-{
-}
+    [OutputElementHint("strong")]
+    public class OutputElementHintTagHelper : TagHelper
+    {
+    }
 
-[HtmlTargetElement("a")]
-[HtmlTargetElement("p")]
-[OutputElementHint("div")]
-public class MultipleDescriptorTagHelperWithOutputElementHint : TagHelper
-{
-}
+    [HtmlTargetElement("a")]
+    [HtmlTargetElement("p")]
+    [OutputElementHint("div")]
+    public class MultipleDescriptorTagHelperWithOutputElementHint : TagHelper
+    {
+    }
 
-public class NonPublicAccessorTagHelper : TagHelper
-{
-    public string ValidAttribute { get; set; }
-    public string InvalidPrivateSetAttribute { get; private set; }
-    public string InvalidPrivateGetAttribute { private get; set; }
-    protected string InvalidProtectedAttribute { get; set; }
-    internal string InvalidInternalAttribute { get; set; }
-    protected internal string InvalidProtectedInternalAttribute { get; set; }
-}
+    public class NonPublicAccessorTagHelper : TagHelper
+    {
+        public string ValidAttribute { get; set; }
+        public string InvalidPrivateSetAttribute { get; private set; }
+        public string InvalidPrivateGetAttribute { private get; set; }
+        protected string InvalidProtectedAttribute { get; set; }
+        internal string InvalidInternalAttribute { get; set; }
+        protected internal string InvalidProtectedInternalAttribute { get; set; }
+    }
 
-public class SingleAttributeTagHelper : TagHelper
-{
-    public int IntAttribute { get; set; }
+    public class SingleAttributeTagHelper : TagHelper
+    {
+        public int IntAttribute { get; set; }
+    }
+
+    public class MissingAccessorTagHelper : TagHelper
+    {
+        public string ValidAttribute { get; set; }
+        public string InvalidNoGetAttribute { set { } }
+        public string InvalidNoSetAttribute { get { return string.Empty; } }
+    }
 }
 
```

Comment:
```suggestion
}
```

Reply:
Not sure that it really conveyed my suggestion in this diff, just saying to remove the trailing newline.